neon_mq_connector/connector.py

```diff
@@ -42,25 +44,45 @@
 
 
 class ConsumerThread(threading.Thread):
+
     """Rabbit MQ Consumer class that aims at providing unified configurable interface for consumer threads"""
     @retry(use_self=True)  # Handle connection failures in case MQ server is still starting up
     def __init__(self, connection_params: pika.ConnectionParameters, queue: str, callback_func: callable,
-                 error_func: callable, auto_ack: bool = True, *args, **kwargs):
+                 error_func: callable, auto_ack: bool = True, queue_reset: bool = False,
+                 queue_exclusive: bool = False,
+                 exchange: str = None,
+                 exchange_reset: bool = False,
+                 exchange_type: str = ExchangeType.direct, *args, **kwargs):
         """
             :param connection_params: pika connection parameters
             :param queue: Desired consuming queue
+            :param queue_exclusive: If declared queue has to exclusive for given channel (deletes with it)
```

Comment:
Minor grammar change:
`Marks declared queue as exclusive to a given channel (deletes with it)`